docs/BUILD.bazel

```diff
@@ -40,6 +40,11 @@ stardoc_with_diff_test(
     bzl_library_target = "//npm/private:npm_link_package",
 )
 
+stardoc_with_diff_test(
+    name = "npm_translate_lock.generated",
```

Comment:
I'm not sure if this is the source, but do we want the link to be "npm_translate_lock.generated" instead of "npm_link_all_packages"?

I think that's confusing for most users won't know (or care) how `npm_link_all_packages` gets generated. If I'm looking for docs on `npm_link_all_packages` that's what I'd expect to see in the sidenav...